### DISCUSSION TITLE
What's a good way to start contributing?

### DISCUSSION THREAD

**Discussion by MilkAndBanana1** [WyattBlue/auto-editor#510] [3. General]
I've been using this project for a very long time and I wish I could contribute something to it especially since there's a few ideas in my head that I would like to try and implement into the project.

For context, I know basic to intermediate Python, but I have never contributed to a project in this scale and I have zero idea how this thing actually works. I looked through the code and it was impossible for me to understand what was happening.

If there is a potential guide or a breakdown of how each component in the project works that I may have missed, or if all of this just comes down to skill issue, let me know so I can see if contributing is even a good idea.

**Comment by WyattBlue** [comment]
Hi, thanks for your interest! It's always a good idea to contribute regardless of skill level.

Making good a bug report, triaging issues, making detailed feature ideas, and answering other peoples questions, are good ways to to start contributing.
